R/zzz.R

```diff
@@ -0,0 +1,31 @@
+
+#' @importFrom rstudioapi isAvailable versionInfo
+#' @importFrom utils sessionInfo
+.onLoad <- function(libname, pkgname) {
+  agent_option <- getOption("worldfootballR.agent")
+  if (is.null(agent_option)) {
+    prefix <- if(interactive() & rstudioapi::isAvailable()) {
+      version_info <- rstudioapi::versionInfo()
+      substr(version_info$mode, 1, 1) <- toupper(substr(version_info$mode, 1, 1))
+      sprintf(
+        "RStudio %s (%s)",
+        version_info$mode,
+        version_info$version
+      )
+    } else {
+      "RStudio Desktop (2022.7.1.554)"
```

Comment:
hard-code an rstudio version, to get things to work (even if the user isn't using RStudio to run their script).